src/pygambit/game.pxi

```diff
@@ -336,6 +337,34 @@ class Game:
         g.title = title
         return g
 
+    def to_arrays(self, dtype=fractions.Fraction) -> typing.List[np.array]:
+        """Generate the payoff tables for players represented as numpy arrays.
+        Parameters
+        ----------
+        dtype : type
```

Comment:
The explanation of this could be a bit more explicit - that it's the type to which payoff values will be converted and the resulting arrays will be of that `dtype`.

Also, we are not testing this parameter at all in the tests yet - either testing for correctness when it should work, and also then testing that it does raise an exception on a type that doesn't work.

Reply:
I added one new test (float) and updated the old one (fraction)